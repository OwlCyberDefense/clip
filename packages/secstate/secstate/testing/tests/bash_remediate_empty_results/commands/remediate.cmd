/usr/sbin/secstate remediate --xccdf-results /root/empty.results.xml
