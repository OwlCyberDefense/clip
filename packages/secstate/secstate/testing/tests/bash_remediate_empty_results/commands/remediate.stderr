/root/empty.results.xml:1: parser error : Extra content at the end of the document

^
Not a valid xccdf result file: /root/empty.results.xml
